synapse/handlers/initial_sync.py

```diff
@@ -433,25 +433,3 @@ def snapshot_all_rooms(
             ret["membership"] = membership
 
         return ret
-
-    async def _check_in_room_or_world_readable(self, room_id, user_id):
```

Comment:
this was an exact cut-and-paste of `Auth.check_in_room_or_world_readable`; removed for sanity.